src/instructlab/lab.py

```diff
@@ -896,40 +895,23 @@ def convert(self, value, param, ctx) -> "torch.device":
 
 @cli.command()
 @click.option("--data-dir", help="Base directory where data is stored.", default=None)
-@click.option(
-    "--input-dir",
-    type=click.Path(),
-    show_default=True,  # TODO: set to None and change help message
-    help="Path to generated files to use as input.",
-)
 @click.option(
     "--skip-preprocessing",
     is_flag=True,
 )
 @click.option(
-    "--tokenizer-dir",
-    help="Base directory where tokenizer is stored.",
-    default=None,
-    show_default=True,
-)
-@click.option(
-    "--gguf-model-path",
-    help="Local directory where gguf model is stored.",
-    default=None,
-    show_default=True,
+    "--input-dir",
+    type=click.Path(),
+    show_default=True,  # TODO: set to None and change help message
+    help="Path to generated files to use as input.",
 )
 @click.option(
-    "--model-dir",
-    help="Base directory where model is stored.",
+    "--model-repo",
+    help="Base repository where model is stored.",
     default="instructlab/merlinite-7b-lab",
     show_default=True,
 )
 @click.option("--iters", help="Number of iterations to train LoRA.", default=100)
-@click.option(
-    "--local",
-    is_flag=True,
-    help="Whether or not `model_dir` is remote from HuggingFace.",
-)
```

Comment:
@cdoern are we sure we want to remove this? maybe we need to have a larger conversation at some point about not forcing users to download a model all over again just for training and allow them to use a model they already have on their system

Reply:
i don't mind. I just found that it was not used as intended in most scenarios. I say we remove for now and add it back in properly

Reply:
How was it being used?

Reply:
well, when I fixed up the code so that --gguf--model-path was removed, I think it was only used when that flag was used? We never really pointed it to local safetensors I don't think.

Reply:
@jaideepr97 you good for removing it or prefer keeping it in?